src/components/entry/Review.js

```diff
@@ -1,11 +1,23 @@
+import { Pencil as PencilIcon } from '@styled-icons/boxicons-solid'
 import styled from 'styled-components/macro'
 
+import { FRUITING_RATINGS, RATINGS } from '../../constants/ratings'
 import ImagePreview from '../ui/ImagePreview'
 import Rating from '../ui/Rating'
+import ResetButton from '../ui/ResetButton'
 import { formatISOString } from './textFormatters'
 
 const ReviewContainer = styled.div`
   margin-bottom: 20px;
+
+  ${({ editable, theme }) =>
```

Comment:
Not sure if React complains about editable being passed to the div but if it does: https://styled-components.com/docs/api#transient-props